packages/models-library/src/models_library/api_schemas_directorv2/comp_tasks.py

```diff
@@ -36,6 +36,10 @@ class ComputationCreate(BaseModel):
         description="the computation shall use the cluster described by its id, 0 is the default cluster",
     )
     simcore_user_agent: str = ""
+    use_on_demand_clusters: bool = Field(
+        default=False,
+        description="if True, a cluster will be created as necessary (wallet_id cannot be None, and cluster_id must be None)",
```

Comment:
Q: the validator below do not check for wallet_id!?

Reply:
no cause at the moment there are none.